viral/ensemble_reactivation.py

```
@@ -0,0 +1,866 @@
+        "pre_shuffled": preactivation_strength_shuffled,
+    }
+
+    processed_matrices = {}
+    for name, matrix in matrices.items():
+        if smooth:
+            # processed = zscore(matrix, axis=1)
+            # processed = gaussian_filter1d(processed, 30)
+            processed = gaussian_filter1d(matrix, 30)
+        else:
+            # processed = zscore(matrix, axis=1)
+            processed = matrix
```

Ruff (0.12.2)

Use ternary operator `processed = gaussian_filter1d(matrix, 30) if smooth else matrix` instead of `if`-`else`-block

(SIM108)